documentation/src/main/asciidoc/topics/con_expiration.adoc

```diff
@@ -0,0 +1,23 @@
+[id='expiration-{context}']
+= Expiration
+Expiration removes entries from memory when they reach one of the following
+time limits:
+
+Lifespan::
+Sets the maximum amount of time that entries can exist.
+
+Maximum idle::
+Specifies how long entries can remain idle. If operations do not occur for
+entries, they become idle.
+
+[IMPORTANT]
+====
+Maximum idle expiration does not currently support:
+
+* Entries stored in off-heap memory.
+* Cache configurations with persistent cache stores.
+
+When using maximum idle expiration with an exception-based eviction policy,
```

Comment:
```suggestion
When using expiration with an exception-based eviction policy,
```

Reply:
I am not sure why the old doc only mentioned that for maximum idle, it is all of expiration.